src/cloud-functions.ts

```diff
@@ -322,9 +333,9 @@ export interface MakeCloudFunctionArgs<EventData> {
   dataConstructor?: (raw: Event) => EventData;
   eventType: string;
   handler?: (data: EventData, context: EventContext) => PromiseLike<any> | any;
-  labels?: { [key: string]: any };
+  labels?: Record<string, string>;
   legacyEventType?: string;
-  options?: { [key: string]: any };
+  options?: DeploymentOptions;
```

Comment:
Same question as above, what if projects don't have `options` that fit into `DeploymentOptions`?

Reply:
Another good point. Fortunately, no one should be calling `makeCloudFunction` directly, and in our public APIs, we make sure that passed options are typed with `DeploymentOption`. e.g.

https://github.com/firebase/firebase-functions/blob/38f1946df4804002be9c585f386c5ae61fd45b10/src/providers/pubsub.ts#L75

Think we should be okay making this change.